### DISCUSSION TITLE
warning: already initialized constant

### DISCUSSION THREAD

**Discussion by effleurager** [Homebrew/discussions#889] [Everyday usage] [answered]
Any brew command I run has several warnings of constant overwriting from the forwardable, json and ostruct gems. Trying to pristine those gems has no effect, and I'm not sure how to check if this issue is related to anything I've done without uninstalling brew itself.

I've got the [config & doctor](https://gist.github.com/effleurager/9cb8fd9f92b99e450d6372e084b78350) command output, as well as the [warnings](https://gist.github.com/effleurager/9740b08ce8f89c5eaa0005cc8855563e) printed on each command here. How can I figure out what's going on?

**Comment by SMillerDev** [comment, marked as answer]
Remove the gems from the system gempath. Homebrew vendors it's own gems so they conflict.

**Reply by jheyduk** [reply]
I have the same problem. How can I remove the gems? I tried with gem uninstall. My MacBook run on `macOS Big Sur 11.2.3`

**Reply by effleurager** [reply]
`sudo` should do you good, otherwise you have a deeper problem with your ruby installation
